### DISCUSSION TITLE
Guidelines and APIs for auto claim-attestation flow for Admins

### DISCUSSION THREAD

**Discussion by radhay-samagra** [Sunbird-RC/community#223] [Issues]
He @tejash-jl @pramodkvarma I was going through the RC docs and was curious, is there any particular implementation, guidelines  for completing the claim attestation flow internally for an admin where an admin/parent adds the attestation data for the children/dependent entities instead of waiting for the actor to raise claims first.

**Comment by pramodkvarma** [comment]
You mean simply attest the user data within the registry by admin proactively? Not sure if there is a workflow built for this yet? @Renuka-S may know. If not you can contribute one in case you are building. Thanks.

**Reply by parthlawate** [reply]
@radhay-samagra So the usecase you are considering is that the actor would not raise claims & the admin would raise claims on behalf of the actor (which is ok i think) ... But would having the same admin also attest the claim be a good idea ? While an audit trail would exit, should attestation not happen from the entity that owns the data the claim is being made about ?

**Reply by pramodkvarma** [reply]
Yeah, I am curious to understand the practical use case on this. Can you explain @radhay-samagra ? Thanks.
